packages/react/test/specs/behaviors/dialogBehavior-test.tsx

```diff
@@ -0,0 +1,20 @@
+import { dialogBehavior } from 'src/lib/accessibility'
+import Button from 'src/components/Button/Button'
```

Comment:
nit: no need to take a dependency on stardust for this test, you can use `<button />`